### ISSUE TITLE
run_execute does not work

### ISSUE THREAD

**Issue by BastianGrahm** [tira-io/tira#143]
![image](https://user-images.githubusercontent.com/29681076/141705850-13ab195d-6fbf-46b5-875a-9b7d0f784b8d.png)
When clicking the run button the application logs to `warnings.log` the following error message.

```
ERROR 2021-11-15 01:28:49,815 actions: /grpc/hyperpartisan-news-detection/test-user-04/run_execute/software1: Server Error: tuple index out of range
Traceback (most recent call last):
  File "/tira/tira-application/src/./tira/actions.py", line 38, in func_wrapper
    response = func(request, *args, **kwargs)
  File "/tira/tira-application/src/./tira/actions.py", line 470, in run_execute
    response = grpc_client.run_execute(vm_id=vm_id,
  File "/tira/tira-application/src/./tira/grpc_client.py", line 35, in func_wrapper
    grpc_transaction = new_transaction(f"initialized {msg} of {args[1]}")
IndexError: tuple index out of range
```

Point of failure is the [`auto_transaction` decorator](https://github.com/tira-io/tira/blob/b6ec7481848e8a61d9d7f33dc727f13396208a34/tira-application/src/tira/grpc_client.py#L35). `args` seems to be empty?

**Comment by kaaage** [comment]
Fixed by 31b8d2407b7c1efeb13c4af7357d18c47a48b5ff

**Comment by kaaage** [comment]
Fixed by 106e62d71198294ad0abe60a8954fcd389d8fd7d
